tests/chaos/oadp/conftest.py

```
@@ -79,3 +81,28 @@ def drain_vm_source_node(admin_client, rhel_vm_with_dv_running, oadp_backup_in_p
     with node_mgmt_console(admin_client=admin_client, node=vm_node, node_mgmt="drain"):
         wait_for_node_schedulable_status(node=vm_node, status=False)
         yield vm_node
+
+
+@pytest.fixture()
```

Ruff (0.14.14)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)